src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/matmul_decompress_convert.cpp

```diff
@@ -485,7 +541,7 @@ class MatMulDecompressConvertTest2 : public MatMulDecompressConvertTest {
         }
         std::shared_ptr<ov::Node> inputWeights =
             ov::test::utils::deprecated::make_constant<float>(weiConstElemType, inShapeWeights.get_shape(), {}, true);
-        if (weiConstElemType == ElementType::f16) {
+        if (weiConstElemType == ElementType::f16 && weiConstElemType != convertOutType) {
```

Comment:
What is the purpose of instoducing `weiConstElemType != convertOutType` condition?